src/ophyd_async/core/signal.py

```diff
@@ -253,6 +253,18 @@ def set_sim_callback(signal: Signal[T], callback: ReadingValueCallback[T]) -> No
     return _sim_backends[signal].set_callback(callback)
 
 
+def create_soft_signal_rw(
```

Comment:
Slight preference for:
```suggestion
def soft_signal_rw(
```